scripts/deployments/config/nonDeployerAccounts.ts

```diff
@@ -0,0 +1,15 @@
+import { ethers } from "hardhat";
+import { NonDeployerConfigAccounts } from "../../../test/helpers/types";
+
+export const nonDeployerConfigAccounts: NonDeployerConfigAccounts = {
+    // Addresses to be replaced with actual values
+    nexusGuardian: "0xf39Fd6e51aad88F6F4ce6aB8827279cffFb92266",
+    rewarderSteward: "0xf39Fd6e51aad88F6F4ce6aB8827279cffFb92266",
+    tavernBarkeeper: "0xf39Fd6e51aad88F6F4ce6aB8827279cffFb92266",
```

Comment:
what are these accounts?

Reply:
placeholder accounts for addresses to be used, changed to deployers address